invenio_sipstore/api.py

```diff
@@ -244,7 +244,9 @@ def create(cls, pid, record, archivable, create_sip_files=True,
         """
         files = record.files if create_sip_files else None
         mtype = SIPMetadataType.get_from_schema(record['$schema'])
-        metadata = {mtype.name: json.dumps(record.dumps())}
+        metadata = {mtype.name: json.dumps(record.dumps(),
+                                           indent=4,
```

Comment:
I did a quick check on a normal sized Zenodo record. Adding indents adds around ~3kb of whitespace (8.4kb to 11.5kb). With 360k SIP packages you're looking at around 1GB of whitespace and increasing storage requirements with about 35% (2.8GB to 3.9GB) just by adding indents.